### PR TITLE
Remove mention of services not being supported

### PR DESCRIPTION
**Pull request by devnoname120**
They are now supported. See: https://github.com/nektos/act/pull/1949

### CODE REVIEW

**Comment by ChristopherHX** [comment]
Services without running the main job in a container as well are still not supported, maybe add this somewhere?

I overall agree, I approve once this get's one approval from someone else

**Review by catthehacker — APPROVED** [review]

**Review by ChristopherHX — APPROVED** [review]

**Pull request merged**
